atom/app/atom_main_delegate.cc

```diff
@@ -156,6 +156,20 @@ base::FilePath InitializeUserDataDir() {
     command_line->AppendSwitchPath(switches::kUserDataDir, user_data_dir);
   }
 
+#if defined(OS_WIN)
```

Comment:
the change we need to make is to use user-data-dir-name instead of user-data-dir and we can pass it just like we passed user-data-dir, but this code is going to conflict and cause issues with crash reporting changes

Reply:
This change didn't require `user-data-dir-name` or `user-data-dir` switch. It will be overwritten if any of these switches present.

Reply:
we can't use user-data-dir like this anymore and this change will conflict with the crash reporting changes and it also doesn't the the value early enough (I am now setting it in atom_main.cc)

Reply:
see https://github.com/brave/muon/pull/279/files#diff-a2a29fbd6a5f9747649d3c2c444dae3fR93
but either way I don't think we should set it based on the exe name. We should use the new `user-data-dir-name` flag or the env var `CHROME_USER_DATA_DIR`

Reply:
@bridiver I think the problem is that on Windows: how would you forcibly set this? The best option we have would be appending that to the Desktop shortcut... but that wouldn't cover all cases of course

Also: is the work @darkdh here for Linux using `user-data-dir` going to break crash reporting?
- https://github.com/brave/electron-installer-debian/pull/1
- https://github.com/brave/electron-installer-redhat/pull/1

Reply:
I will change all the `user-data-dir` to `user-data-dir-name` after https://github.com/brave/muon/pull/279 merged